discovery-provider/alembic/versions/c967ae0fcaf6_add_aggregate_metrics_tables.py

```diff
@@ -0,0 +1,78 @@
+"""Add aggregate metrics tables
+
+Revision ID: c967ae0fcaf6
+Revises: 579360c7cbf3
+Create Date: 2021-02-24 17:45:45.939947
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'c967ae0fcaf6'
+down_revision = '5dd6a55bb738'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table('daily_unique_users_metrics',
```

Comment:
if "aggregate" is the wording we're using for all the discovery nodes together, i'd suggest putting that in the table names and stuff too. cause otherwise i'm sure we'll get confused between app_name_metrics and daily_app_name_metrics. conversely we could rename the existing app_name_metrics and route_metrics as app_name_metrics_single_node or something like that

Reply:
100%